docs/topic-guides/blendable-web-apps/common-look-and-feel.md

```diff
@@ -0,0 +1,84 @@
+
+## Mental model of client-side rendering by the Starcounter host
+
+Starcounter host follows a standards-based layered approach that separates the responsibilities of the app author, the design system, and the solution owner. 
+
+The full design consists of:
+
+1. Content defined by native HTML
+2. Common UI patterns and components of Uniform design system
+3. Theme configuration through custom CSS properties
+4. Extend or replace the theme with HTML, CSS, and JavaScript
+
+### Content defined by native HTML
+
+As an app author, you provide the views for your apps with a clear separation of the content and the presentation.
+
+In the content part, you provide the native semantic HTML elements that can be rendered on the screen directly or with their appearance changed in the presentation layer.
+
+### Common UI patterns and components of Uniform design system
+
+As the basic way for providing a common look and feel in the presentation layer, Starcounter provides Uniform design system. It consists of custom CSS properties, CSS reset and a library of patterns, and components.
+
+Uniform follows the inversion of control principle, which makes it a link between app authors and solution owners.
+
+Unlike many design systems, Uniform is open-ended and themeable. It can be configured, extended or replaced in parts to achieve anything that is possible with HTML, CSS, and JavaScript.
+
+The custom CSS properties are the basic set of primitives. They define the fonts, sizes, colors, margins and other values used throughout the reset, patterns and components.
+
+The CSS reset overwrites the default user agent stylesheet. It provides the look of the native HTML5 elements such as the headings (H1-H6), links, inputs, buttons, and tables. It is implicitly loaded by the Starcounter app shell, so there is no need to load it explicitly in the views. 
+
+The pattern library, [Uniform.css](https://starcounter.github.io/uniform.css/), is a set of CSS classes that implement visual hierarchy and give specific roles to native elements. It provides sections, cards, titles, column layouts, alerts,  buttons, etc.
+
+The component library, [Uniform components](https://starcounter.github.io/uniform.css/components/), is a set of custom elements that implement interactive UI controls. For the beginning, we provide a form item, a date picker, a data table and a pagination control. 
+
+To use the patterns and the components, you must import them explicitly. They are placed in the shadow root of the view container to wrap the distributed content elements. That allows to serve real progressive enhancement, and what is crucial for a solution of blended apps: keep app content's markup semantic and unopinionated, giving solution owner a way to apply own opinion without changing the app source code.
```

Comment:
> They are placed in the shadow root of the view container to wrap the distributed content elements.

This sentence is Greek to me. I have no idea what it's supposed to mean. 

Overall, this whole paragraph is confusing to me. 
What's "real progressive enhancement"? What are you trying to say with "and what is crucial for a solution of blended apps:"?

Can you try to explain this section in other words to me and see if we can rephrase it?

Reply:
I agree this is too technical. I propose to rephrase it as:

To use the Uniform stylesheet and components, you must import them explicitly in the presentation layer of the view.